.vscode/launch.json

```diff
@@ -16,6 +16,22 @@
 			],
 			"preLaunchTask": "watch"
 
+		},
+		{
+			"name": "Launch Web Client",

```

Comment:
I don't think we want to go that far yet, but we could have an integration test with https://github.com/microsoft/vscode-test-web